recipes/sidetable/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sidetable" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 44695ab38e7e166664bf79cf5af23f4360f4d9a5e17114b5ebc4a6908f8be3d9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pandas >=1.0
```

Comment:
Do you need pandas to build this package?

Reply:
No. To use it yes but not to build it. I will fix and re-submit.